### PR TITLE
change version to 1.7.0

### PR DESCRIPTION
**Pull request by rogertsui0923**

### CODE REVIEW

**Comment by rogertsui0923** [comment]
@asoesilo This PR just updates the version number. After Afian's review I will make another PR from `models-update` to `master`. All of that code has been reviewed before merging with the feature branch; does it need to be re-reviewed going into master?

**Comment by asoesilo** [comment]
>All of that code has been reviewed before merging with the feature branch; does it need to be re-reviewed going into master?

Can you tag me as a reviewer before merging it into master? Thanks!

File: routific.gemspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Gem::Specification.new do |s|
   s.name              = 'routific'
-  s.version           = '1.1.2'
+  s.version           = '1.7.0'
   s.date              = '2017-06-26'
```
Comment (asoesilo) [inline routific.gemspec]: Let's also change the `date` to `2017-09-21`.

Can you also remove `s.authors` and update `s.email` to `support@routific.com`?

Comment (rogertsui0923) [reply routific.gemspec]: Sure!

Comment (rogertsui0923) [reply routific.gemspec]: Hmm, apparently `authors may not be empty`

**Comment by rogertsui0923** [comment]
@asoesilo I could not build the gem without authors, so I've just set it to "Routific". If that's okay with you, ready for another review!

**Pull request merged**